src/dbos-executor.ts

```diff
@@ -928,6 +932,13 @@ export class DBOSExecutor implements DBOSExecutorContext {
     return handlerArray;
   }
 
+  async deactivateConsumers() {
```

Comment:
Can we add a test for this method?

Reply:
Maybe not for Kafka specifically, but for scheduled workflows (test that no more start), which should be easier?